### ISSUE TITLE
English support?

### ISSUE THREAD

**Issue by so77id** [ethan-funny/explore-python#1]

**Comment by ethan-funny** [comment]
I'm sorry that I have no plans to write the English version. Thanks.

**Comment by so77id** [comment]
Wait for some super hero to help, thanks 😆

**Comment by imGurpreetSK** [comment]
Hi,
Can I contribute by translating the text to English?
Thanks.

**Comment by ethan-funny** [comment]
@GurpreetSK95 Yes, you can translate the text to English. And please let me know if you finish translating or need some help. Thanks.

**Comment by MDShahrouq** [comment]
Please do translate in English, it could benefit a lot of people @ethan-funny

**Comment by MDShahrouq** [comment]
@GurpreetSK95 If you can translate that it would be better

**Comment by imGurpreetSK** [comment]
@MDShahrouq I could not start contributing this past week due to some work issues. Will start this week, one at a time :D

**Comment by so77id** [comment]
ty @GurpreetSK95  you are the best :)
